src/app/components/navigators/MainNavigator.tsx

```diff
@@ -8,8 +8,8 @@ const Stack = createStackNavigator();
 const MainNavigator: React.FC = () => (
   <Stack.Navigator
     initialRouteName="Tabs"
-    headerMode="none"
     screenOptions={{
+      headerShown: false,
```

Comment:
wat doet dit precies?

Reply:
Een stack navigator heeft standaard een header, die een titel toont. Maar is deze stack navigation zit ook een tab navigator met óók een header. Hierdoor zou je een dubbele header krijgen. Dit hide de header. Was al zo met `    headerMode="none"`, maar de API van React Navigation is veranderd